suites/reef/rbd/tier-2_rbd_mirror_metrics.yaml

```diff
@@ -180,3 +178,17 @@ tests:
             io_total: 4G
       polarion-id: CEPH-83575564
       desc: Validate rbd mirror daemon related journal based metrics
+
+  - test:
+      name: Test to verify rbd mirror related network connection metrics
+      module: test_rbd_mirror_connection_metrics.py
```

Comment:
Are we covering this test case only for replicated pool? Is it not applicable for ecpool?

Reply:
it doesn't matter for rep pool or EC pool it's just test of metrics network health connections.

Reply:
the initial_mirror_config method is implemented only for replicated pools is it? We don't have ec pools included in it? 
In initial_rbd_config we have ec pools also included right?

Reply:
yes `initial_mirror_config` it creates mirroring for replicated pool,
for EC pool inclusion we need to use `rbd_mirror_config`

Reply:
As discussed since you are very sure that changing the pool type will not have any impact on the metrics I am going ahead and approving this PR now.